### ISSUE TITLE
[fashion-MNIST, task 4] KeyError while submitting the solution

### ISSUE THREAD

**Issue by rafajak** [neptune-ml/minerva-training-materials#69]
neptune run -- submit --problem fashion_mnist --task_nr 4

```
-- | --

5.372691 | /mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
5.372979 | from ._conv import register_converters as _register_converters
5.373169 | Using TensorFlow backend.
12.010692 | [NbConvertApp] Converting notebook /mnt/ml-team/homes/rafal.jakubanis/minerva/resources/fashion_mnist/tasks/task4.ipynb to python
12.892425 | [NbConvertApp] Writing 641 bytes to /mnt/ml-team/homes/rafal.jakubanis/minerva/resources/fashion_mnist/tasks/task4.py
13.172734 | Traceback (most recent call last):
13.173071 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/deepsense/neptune/job_wrapper.py", line 103, in execute
13.17331 | execfile(job_filepath, job_globals)
13.17354 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/past/builtins/misc.py", line 82, in execfile
13.173774 | exec_(code, myglobals, mylocals)
13.174001 | File "main.py", line 66, in <module>
13.174223 | action()
13.174454 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/click/core.py", line 722, in __call__
13.174719 | return self.main(*args, **kwargs)
13.174942 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_ve
nv/lib/python3.5/site-packages/click/core.py", line 697, in main
13.175152 | rv = self.invoke(ctx)
13.175377 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
13.175583 | return _process_result(sub_ctx.command.invoke(sub_ctx))
13.175779 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/click/core.py", line 895, in invoke
13.17597 | return ctx.invoke(self.callback, **ctx.params)
13.17616 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/click/core.py", line 535, in invoke
13.176348 | return callback(*args, **kwargs)
13.176538 | File "main.py", line 61, in submit
13.176747 | pm.submit_task(task_sub_problem, task_nr, file_path, dev_mode)
13.176932 | File "/mnt/ml-team/homes/rafal.jakubanis/minerva/minerva/fashion_mnist/problem_manager.py", line 34, in submit_task
13.177115 | new_trainer = task_handler.substitute(user_task_solution, user_config)
13.177299 | File "/mnt/ml-team/homes/rafal.jakubanis/minerva/minerva/backend/task_manager.py", line 14, in substitute
13.177482 | self.modify_trainer(user_solution, user_config)
13.177665 | File "/mnt/ml-team/homes/rafal.jakubanis/minerva/minerva/fashion_mnist/tasks.py", line 54, in modify_trainer
13.177847 | self.trainer.pipeline.get_step('loader').is_substituted = True
13.17803 | File "/mnt/ml-team/homes/rafal.jakubanis/minerva/minerva/backend/base.py", line 54, in get_step
13.178213 | return self.all_steps[name]
13.178396 | KeyError: 'loader'
13.178578 |  
13.178761 | During handling of the above exception, another exception occurred:
13.178949 |  
13.179133 | Traceback (most recent call last):
13.179326 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/deepsense/neptune/job_wrapper.py", line 109, in <module>
13.17951 | execute()
13.179693 | File "/mnt/ml-team/homes/rafal.jakubanis/envs/minerva_venv/lib/python3.5/site-packages/deepsense/neptune/job_wrapper.py", line 105, in execute
13.179914 | raise ExperimentExecutionException("Exception during experiment execution", ex)
13.180092 | deepsense.neptune.exceptions.ExperimentExecutionException: ('Exception during experiment execution', KeyError('loader',))
 


```

**Comment by kamil-kaczmarek** [comment]
@rafajak thank for the issue. I'll take a look at it

**Comment by kamil-kaczmarek** [comment]
@rafajak it is fixed in the #72
